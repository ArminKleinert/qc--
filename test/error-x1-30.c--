
register bits1 one;

foo() {
    return <one / 1>;
}
