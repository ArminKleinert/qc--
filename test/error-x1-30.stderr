file error-x1-30.c--, line 5, characters 13-16 Error: access of non-constant one
